src/intl/en/page-eth.json

```diff
@@ -55,6 +55,7 @@
   "page-eth-underpins": "ETH underpins the Ethereum financial system",
   "page-eth-underpins-desc": "Not satisfied with payments, the Ethereum community is building a whole financial system that's <a href=\"/glossary/#peer-to-peer-network\">peer-to-peer</a> and accessible to everyone.",
   "page-eth-underpins-desc-2": "You can use ETH as collateral to generate entirely different cryptocurrency tokens on Ethereum. Plus you can borrow, lend and earn interest on ETH and other ETH-backed tokens.",
+  "page-eth-weth": "Wrapped ether (WETH) is used to extend the functionality of ETH to work with other tokens and applications. <a href=\"/wrapped-eth\">Learn more about WETH</a>.",
```

Comment:
<img width="1013" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/834101e3-f78c-4b59-b61f-349c900a9c50">
Actually, small bug still.. will get this patched

Reply:
Fixed with 2bcb9c6